### DISCUSSION TITLE
getting error and i cant locate zksync sepolia on my metamask so i cant deploy to zksync

### DISCUSSION THREAD

**Discussion by munem760** [Cyfrin/foundry-full-course-cu#2802] [General]
Error: "underlying network changed (event=\"changed\", network={\"chainId\":300,\"name\":\"zkSync Sepolia Testnet\"}, detectedNetwork={\"name\":\"sepolia\",\"chainId\":11155111,\"ensAddress\":null}, code=NETWORK_ERROR, version=providers/5.7.2)"

**Comment by EngrPips** [comment]
You are currently connected to the Ethereum sepolia chain in your `Metamask` You need to switch your `Metamask` network to ZKSync sepolia and then retry. Please ask questions if anything is still confusing.
